### DISCUSSION TITLE
建议移除”发弹幕自动佩戴粉丝勋章“功能

### DISCUSSION THREAD

**Discussion by ADJazzzz** [andywang425/BLTH#501] [Development]
在新版直播间中，官方添加了`优先展示当前直播间的粉丝勋章`功能，即进入直播间自动佩戴相应勋章，与本脚本的`发弹幕自动佩戴粉丝勋章`功能相对应，而且比本脚本的更加方便，可以考虑移除掉
![image](https://github.com/andywang425/BLTH/assets/68748165/f404dca6-170a-4e3c-b8e8-a411268d8d3d)

**Comment by ADJazzzz** [comment]
@andywang425

**Comment by andywang425** [comment]
确实可以移除，官方可算做这个功能了
